app/models/result_validation_form.rb

```diff
@@ -9,7 +9,6 @@ class ResultValidationForm
 
   COMP_VALIDATION_MODES = [["Pick competition(s) manually", COMP_VALIDATION_MANUAL], ["Execute for ALL competitions", COMP_VALIDATION_ALL]].freeze
 
-  ALL_COMPETITIONS_SCOPE = Competition.over.not_cancelled
```

Comment:
How did this even work. Wouldn't that variable only contain Competitions that were over/not_cancelled on boot?

Reply:
No, actually Rails scopes are lazyily.
The thing that was failing was not the loading of actual data but the basic model validations like "which columns exist" etc. Rails wants to execute those as soon as you speak the name of any model, even if you're not fetching data.